web-mode/vue

```diff
@@ -0,0 +1,18 @@
+# -*- mode: snippet -*-
+# name: vue
+# uuid: vue
+# key: vue
+# condition: t
+# --
+<script>
+  export default {
+    name: '${0:`(f-base buffer-file-name)`}'
+  }
+</script>
+
+<template>
+  $1
```

Comment:
Replace `$1` with `${1:%}`. This way, if a selection is active when the snippet is inserted, it'll be inserted at that tab stop.

Reply:
I fail to understand this change. I've created an empty file, typed `<p>Testing</p>` in it, went to visual mode, selected it, then inserted the snippet. All I saw at this position was `%`. What am I missing?

Reply:
Ah, my bad, it was meant to be ```${1:`%`}``` (note the backticks)